CONP_DATS_fields.md

```diff
@@ -25,19 +25,19 @@ We use the `DATS.json` file format to store metadata describing CONP datasets.
+|Parent dataset id|extraProperties - parent_dataset_id|**REQUIRED IF** the dataset is a derived dataset, the parent dataset id (as specified in _conp_dataset/.gitmodules_).|Identifier (DOI) of the source dataset used to generate this dataset.|
 |Primary Publications|primaryPublications|**RECOMMENDED** The primary publication(s) associated with the dataset, usually describing how the dataset was produced.| The primary publication(s) associated with the dataset, usually describing how the dataset was produced.|
-|Title|Primary Publications -> Title| The name of the publication, usually one sentence or short description of the publication.|The name of the publication.|
-|Publication Venue|Primary Publications -> Publication Venue|The name of the publication venue where the document is published (if applicable).|The name of the publication venue where the document is published (if applicable).|
-|Authors|Primary Publications -> Authors|Authors of the publication.|Authors of the publication.|
-|Affiliations|Primary Publications -> Authors -> Affiliations|Author affiliations.|Author affiliations.|
-|Date|Primary Publications -> Dates|Relevant dates for the publication. If you provide a date, it must come with a description of the date.|Relevant dates for the publication. If you provide a date, it must come with a description of the date (i.e.: first submission, final approval, date of publication, ...).|
-|Identifier|Primary Publications -> Identifier -> Identifier|**RECOMMENDED** A code uniquely identifying an entity locally to a system or globally.|A code uniquely identifying the publication locally to a system or globally. Provide a *Document Object Identifier (DOI)* if you have one.|
-|Identifier Source|Primary Publications -> Identifier -> Identifier Source|**RECOMMENDED** Information about the organisation/namespace responsible for minting the identifier.| Information about the organisation/namespace responsible for minting the identifier. It must be provided if the identifier is provided.|
+|Title|primaryPublications - title| The name of the publication, usually one sentence or short description of the publication.|The name of the publication.|
+|Publication Venue|primaryPublications - publicationVenue|The name of the publication venue where the document is published (if applicable).|The name of the publication venue where the document is published (if applicable).|
+|Authors|primaryPublications - authors|Authors of the publication.|Authors of the publication.|
+|Affiliations|primaryPublications - authors - affiliations|Author affiliations.|Author affiliations.|
+|Date|primaryPublications -> dates|Relevant dates for the publication. If you provide a date, it must come with a description of the date.|Relevant dates for the publication. If you provide a date, it must come with a description of the date (i.e.: first submission, final approval, date of publication, ...).|
```

Comment:
is there a reason why some have `-` and some have `->`? If not, could this be harmonized? I still see several with the arrow instead of the dash in the PR.

Reply:
No particular reason, and I have just updated the remaining arrows to dashes now.